modules/understanding-upgrade-channels.adoc

```diff
@@ -7,116 +7,99 @@
 ====
 
+== candidate-{product-version} channel
+
+The `candidate-{product-version}` channel contains releases as soon as they're produced, prior to being declared GA. Releases present only in the candidate channels have not completed full Quality Assurance testing and lack Red Hat Support. Therefore releases present only in the candidate channels are only suitable for testing purposes where destroying and recreating a cluster is acceptable. Additionally, we provide no guarantee that these releases will be offered update paths to GA versions.
+
+ifdef::openshift-origin[]
+[id="stable-4-channel_{context}"]
+== stable-4 channel
+Releases are added to the `stable-4` channel after passing all tests and stable-4 is the only supported channel.
 endif::openshift-origin[]
 
+
+
 [id="upgrade-version-paths_{context}"]
-== Upgrade version paths
+== Update version paths
 
-{product-title} maintains an upgrade recommendation service that understands the version of {product-title} you have installed as well as the path to take within the channel you choose to get you to the next release.
+{product-title} maintains an update recommendation service that understands the version of {product-title} you have installed as well as the path to take within the channel you choose to get you to the next release. Update paths are also limited to versions relevant to your currently selected channel and its promotion characteristics.
 
-ifndef::openshift-origin[]
-You can imagine seeing the following in the `fast-{product-version}` channel:
-endif::openshift-origin[]
-ifdef::openshift-origin[]
-You can imagine seeing the following in the `stable-4` channel:
-endif::openshift-origin[]
+You can imagine seeing the following releases in your channel:
 
 * {product-version}.0
 * {product-version}.1
 * {product-version}.3
 * {product-version}.4
 
-The service recommends only upgrades that have been tested and have no serious issues. It will not suggest updating to a version of {product-title} that contains known vulnerabilities. For example, if your cluster is on {product-version}.1 and {product-title} suggests {product-version}.4, then it is safe for you to update from {product-version}.1 to {product-version}.4. Do not rely on consecutive patch numbers. In this example, {product-version}.2 is not and never was available in the channel.
+The service recommends only updates that have been tested and have no known serious regressions. For example, if your cluster is on {product-version}.1 and {product-title} suggests {product-version}.4, then it is safe for you to update from {product-version}.1 to {product-version}.4. Do not rely on consecutive patch numbers. In this example, {product-version}.2 is not and never was available in the channel, therefore updates to {product-version}.2 are neither recommended nor supported.
```

Comment:
`{product-version}.2 is not and never was available in the channel, therefore updates to {product-version}.2 are neither recommended nor supported.` has some caveat. We should say ` if {product-version}.2 is not and never was released and not available in the channel, therefore updates to {product-version}.2 are neither recommended nor supported.`

Reply:
this section covers `candidate-*` too, so "then it is safe" -> "then it is recommended"?  Or something else that leaves more room for "given the stability commitments of your configured channel".

Reply:
@wking ok, will make that change. I'd also considered whether or not we need to get into this much detail here. If the cluster is connected and offering updates then they're recommended, in this context they'll never know that "{product=version}.2" existed and I'm not too worried about the risk of them hammering that screw in.